### PR TITLE
Server - Add display name in GetFeatureInfo response

### PR DESCRIPTION
**Pull request by Gustry**
Add the display name in GetFeatureInfo response if needed thanks to `WITH_DISPLAY_NAME`, similar to the existing `WITH_MAPTIP`.

Funded by 3liz.com https://3liz.com

### CODE REVIEW

**Review by rldhont — APPROVED** [review]

**Comment by Gustry** [comment]
It's not stale, waiting for a review/merge. Thanks

File: src/server/services/wms/qgswmsparameters.h [hunk 1]
```diff
@@ -1287,6 +1288,12 @@ namespace QgsWms
        */
       bool withMapTip() const;
 
+      /**
+       * \brief withDisplayName
+       * \returns TRUE if the display name is requested for feature info response
+       */
+      bool withDisplayName() const;
```
Comment (elpaso) [inline src/server/services/wms/qgswmsparameters.h:1295]: \since QGIS x.y

**Review by elpaso — APPROVED** [review]

**Comment by Gustry** [comment]
Thanks @elpaso !

**Comment by Gustry** [comment]
The cppcheck seems failing on master : https://github.com/qgis/QGIS/commits/master
So it's a false positive ?

@rldhont can't merge when there is a failing workflow. @elpaso OK for you to merge ?

**Pull request merged**